app/views/admin/events/_form.html.haml

```diff
@@ -0,0 +1,77 @@
+.row
+  .col-md-12
+    .page-header
+      %h1
+        New Event
+.row
+  .col-md-12
+    = semantic_form_for(@event, url: @url) do |f|
+      = f.inputs name: 'Event Information' do
```

Comment:
Yes, they could be on the beginning, absolutely. It's probably even gonna look better if we add the `= semantic_form_for` in admin/events/_form and then add the speaker/submitter fields and then show the rest of the form from `= render partial: 'proposals/proposal_form', locals: { f: f }`